apps/components-e2e/src/components/combobox/combobox.e2e.ts

```diff
@@ -97,7 +97,8 @@ test('should filter the correct values and not rewrite the input', async (testCo
     .contains('Value 2');
 });
 
-test('should not reset the value when hitting continuously typing the backspace key', async (testController: TestController) => {
+// tslint:disable-next-line: dt-no-focused-tests
+test.skip('should not reset the value when hitting continuously typing the backspace key', async (testController: TestController) => {
```

Comment:
I am perfectly fine with the skip here, one idea maybe worth trying is to have a different timing in the combobox.ts in the e2e app than the wait time in the test. 
The wait here and the timer in 
https://github.com/dynatrace-oss/barista/blob/master/apps/components-e2e/src/components/combobox/combobox.ts#L77

both currently use 1500ms for the delay.
Reduce the wait time in the combobox.ts might make it less flaky

Reply:
Thanks fabian, that helped. 
I will reword the commit to match the solution.